.gitlab-ci.yml

```diff
@@ -0,0 +1,68 @@
+include:
+  - project: hpc/gitlab-pipelines
+    file: spack-build-components.gitlab-ci.yml
+
+# Configure when the pipeline actually runs.
+workflow:
+  rules:
+    # Don't run on push events for now to try and save resources
+    - if: '$CI_PIPELINE_SOURCE == "push"'
+      when: never
+    # Don't run on MR events because we also get "external_pull_request_event" from GitHub.
+    - if: '$CI_PIPELINE_SOURCE == "merge_request_event"'
+      when: never
+    - when: always
```

Comment:
just curious : this would be the case for all GitHub projects I assume? if so, could this be added somewhere else and inherited from there?

Reply:
Good suggestion, done in c1fd1dd9fd41e4588088265bf0da9e41578f34e2 -- unfortunately this introduces another TODO; wait for an external MR to be merged.